taz.de.js

```diff
@@ -1,198 +1,152 @@
+		items[href] = title;
 	}
-	author = author.replace(/^\s*|\s*$/g, '');
-	author = author.replace(".", ". "); // in case a space is missing.
-	author = author.replace("VON ", '');
-	author = author.replace(/\s+/g, ' ');
-	author = author.split(/\sund\s|\su\.\s|\,\s|\&/); 
-	for (var i in author) {
-		if (author[i].match(/\s/)) { // only names that contain a space!
-			author[i] = author[i].replace(/^\s*|\s*$/g, '');
-			author[i] = authorCase(author[i]);
-			newItem.creators.push(Zotero.Utilities.cleanAuthor(author[i], "author"));
-		}
-	}
-	
-	
-	
-	// Section
-	var section_XPath = ".//*[contains(@class, 'selected')]/ul/li[contains(@class, 'selected')]";
-	if (doc.evaluate(section_XPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext() ) {
-		var section= doc.evaluate(section_XPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
-		newItem.section = section;
-	}
-	
-	// Date 
-	var date_XPath = '//div[contains(@class, "sectbody")]/span[@class="date"]';
-	var date = ZU.xpathText(doc, date_XPath);
-	if (date) newItem.date = ZU.trimInternal(date);	
-
-	newItem.attachments.push({url:doc.location.href, title:doc.title, mimeType:"text/html"});
-	newItem.publicationTitle = "die tageszeitung"
-
-	newItem.complete();
-	
+	return found ? items : false;
 }
 
 
 function doWeb(doc, url) {
 	if (detectWeb(doc, url) == "multiple") {
-		var items = new Object();
-		var taz_Multiple_XPath = '//div[contains(@class, "first_page")]//a[h4]';
-		var taz_Search_XPath = '//div[contains(@class, "searchresults")]//a[h4]'   
-
-		 if (doc.evaluate(taz_Multiple_XPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext() ){ 
-			var titles = doc.evaluate(taz_Multiple_XPath, doc, null, XPathResult.ANY_TYPE, null);
-		}  else if (doc.evaluate(taz_Search_XPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext() ){ 
-			var titles = doc.evaluate(taz_Search_XPath, doc, null, XPathResult.ANY_TYPE, null);
-		}
-
-		
-		var next_title;
-		while (next_title = titles.iterateNext()) {
-			items[next_title.href] = next_title.innerHTML;
-			items[next_title.href] = items[next_title.href].replace(/(\<h4.*?\>.*?\<\/h4\>\<h3.*?\>)(.*)\<\/h3\>.*/, '$2');
-		}
-		items = Zotero.selectItems(items, function(items) {
-			if(!items) return true;
-
-			var articles = new Array();
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
 			for (var i in items) {
 				articles.push(i);
 			}
-			ZU.processDocuments(articles, function(doc) { scrape(doc, doc.location.href) });
+			ZU.processDocuments(articles, scrape);
 		});
 	} else {
 		scrape(doc, url);
 	}
-}	
+}
+
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	//translator.setDocument(doc);
+	
+	translator.setHandler('itemDone', function (obj, item) {
+		item.publicationTitle = "taz. die tageszeitung";
```

Comment:
In the ZDB we have `Die Tageszeitung : taz` which looks nice for me. I will change it to this.